pkg/host/store/store.go

```diff
@@ -47,7 +47,7 @@ func createOperatorConfigFolderIfNeeded() error {
 	_, err := os.Stat(SriovConfBasePathUse)
 	if err != nil {
 		if os.IsNotExist(err) {
-			err = os.MkdirAll(SriovConfBasePathUse, os.ModeDir)
+			err = os.MkdirAll(SriovConfBasePathUse, 0777)
```

Comment:
why was this change needed ?

Reply:
In the unit-tests I was not able to write when we create a folder inside the temp folder creation